auth/internal/delivery/grpc/refresh.go

```diff
@@ -0,0 +1,26 @@
+package auth_grpc
+
+import (
+	"context"
+	usecase_dto "msa_big_tech/auth/internal/usecases/dto"
+	auth "msa_big_tech/auth/pkg/proto/v1"
```

Comment:
go fmt в IDE настроить бы для импортов

```suggestion
	"context"
	
	usecase_dto "msa_big_tech/auth/internal/usecases/dto"
	auth "msa_big_tech/auth/pkg/proto/v1"
```